blueocean-dashboard/src/main/js/components/PullRequests.jsx

```diff
@@ -48,7 +48,7 @@ EmptyState.propTypes = {
 
 export class PullRequests extends Component {
     componentWillMount() {
-        if (this.context.config && this.context.params) {
+        if (this.context.pipeline && this.context.params && !pipelineBranchesUnsupported(this.context.pipeline)) {
```

Comment:
... or this, at *mount* time, data is likely to be fetched and is still `$pending`